Robot.sublime-settings

```diff
@@ -74,7 +74,7 @@
         and in Windows this could be like: C:\\Python27\\python.exe
     */
 
-    "path_to_python": "/usr/bin/python",
+    "path_to_python3": "/usr/bin/python",

```

Comment:
Is this change backward compatible? Did you test the upgrade path for those who already have the plugin installed?

Reply:
It won't backward compatible and the upgrade won't work.

Reply:
I can make it back path_to_python and check the python version in setting.py instead. What do you think?

Reply:
That would be better. Also it might be good to mention this in the README

Reply:
I tried to modify the readme but I couldn't. Probably @andriyko  need to do it himself.

Reply:
I have pushed another commit to display an error message if it's not python3